### ISSUE TITLE
Bulbar conjunctival location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11554]
Preferred term label:
Bulbar conjunctival location

Synonyms

    Bulbar location

    Conjunctiva, bulbar

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization referring to the bulbar conjunctiva, the clear mucous membrane that covers the anterior sclera (the globe of the eye) up to the corneoscleral limbus. Used to localize phenotypes such as conjunctival chemosis, pinguecula, or Bitot's spots. PMID:18427514

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Your nano-attribution (ORCID)
0000-0001-9042-0508
